SAMtags.tex

```diff
@@ -662,6 +665,9 @@ \section{Tag History}
 \setlength{\parindent}{0pt}
 \newcommand*{\gap}{\vspace*{2ex}}
 
+\subsubsection*{December 2021}
+Migrate the draft Mm and Ml tags to the official MM and ML namespace.
```

Comment:
I'm glad a history entry has been added, but I think it needs to be written to be more comprehensible and actionable for spec readers than this sentence. Something (in addition perhaps) that spells out “Base modifications should be written using the specification's MM and ML tags, rather than the previous draft Mm and Ml tags“, rather than using only standardese here.

(Also need to update to `\subsubsection*{January 2022}` or whenever this gets merged.)